src/libraries/System.Private.CoreLib/src/System/SpanHelpers.Char.cs

```diff
@@ -2017,5 +2017,96 @@ private static bool TryFindFirstMatchedLane(Vector128<ushort> compareResult, ref
             matchedLane = BitOperations.TrailingZeroCount(selectedLanes) >> 3;
             return true;
         }
+
+        public static void Reverse(ref char buf, nuint length)
```

Comment:
Same general comments as for `SpanHelpers.Byte.cs` apply.

Reply:
Updated.